src/compat/eme/custom_media_keys/webkit_media_keys.ts

```diff
@@ -220,8 +221,21 @@ export default function getWebKitMediaKeysCallbacks() {
   }
   const isTypeSupported = WebKitMediaKeysConstructor.isTypeSupported;
   const createCustomMediaKeys = (keyType: string) => new WebKitCustomMediaKeys(keyType);
+  const customSetMediaKeys = (elt: HTMLMediaElement,
+                              mediaKeys: ICustomMediaKeys|null): void => {
+    if (mediaKeys === null) {
+      if ((elt as any).webkitSetMediaKeys === undefined) {
+        throw new Error("No webkitSetMediaKeys method.");
+      }
+      /* tslint:disable no-unsafe-any */
+      return (elt as any).webkitSetMediaKeys(null);
+      /* tslint:enable no-unsafe-any */
+    }
+    return mediaKeys._setVideo(elt);
```

Comment:
Considering  that _setVideo also calls `webkitSetMediaKeys`, this code could be simplified

Reply:
The problem is that here, `mediaKeys` has an inner native `mediaKeys`, that shall be attached to video element. As the custom `mediaKeys` can be `null`, i find it difficult to imagine a simplification.

Reply:
Ok I understand